src/components/grid-view/GridViewSlider.js

```diff
@@ -40,6 +40,7 @@ const GridViewSlider = ({ columnCount, gridMaxColumns, gridMinColumns, intl, max
                     <IconMinusThin color={bdlGray50} width={14} height={14} />
                 </PlainButton>
                 <input
+                    aria-label={intl.formatMessage(messages.gridViewColumnSize)}
```

Comment:
not necessary but there could be an opportunity to destruct `intl` at the top of the function:
`const { formatMessage } = intl;`
since `formatMessage` is being used in several places